test/fluree/db/reasoner/owl2rl_test.clj

```diff
@@ -288,7 +301,7 @@
                                        "owl" "http://www.w3.org/2002/07/owl#"}
                            "@id"      "ex:mother"
                            "@type"    ["owl:ObjectProperty" "owl:FunctionalProperty"]}])]
-
+      
```

Comment:
I try to keep an eye out for whitespace-only changes like this and avoid staging them - it just adds noise to the git history. Not a big deal, but something to keep in mind in the future.